src/library/scala/collection/SeqLike.scala

```diff
@@ -603,8 +603,13 @@ trait SeqLike[+A, +Repr] extends Any with IterableLike[A, Repr] with GenSeqLike[
 
   /** Sorts this $Coll according to the Ordering which results from transforming
    *  an implicitly given Ordering with a transformation function.
-   *  @see [[scala.math.Ordering]]
    *  $willNotTerminateInf
+   *
+   *  The sort is stable. That is, elements that are equal (as determined by
+   *  `lt`) appear in the same order in the sorted sequence as in the original.
```

Comment:
`lt` is not in scope. Do you mean ord.compare/lt etc

Reply:
Yes - I'm admittedly not aware of a different lt unless you're referring to the parameter passed to sortWith? I noted that both sorted and sortWith had this exact comment in their documentation and from there this felt like the appropriate choice of words.

Reply:
It's a incautious copy paste error in the doc of `sorted` and should be fixed.

Reply:
Ah, my bad then. I've updated the scaladoc to refer to `ord.compare` for both `sortBy` and `sorted`, which I believe should now make the docs correct.